docs/source/admin/fs/rest.rst

```diff
@@ -251,6 +251,17 @@ The field ``external`` doesn't necessarily be a flat structure. This is a more a
 
 .. attention:: Only standard :ref:`FSCrawler fields <generated_fields>` can be set outside ``external`` field name.
 
+Specifying an elasticsearch index
+~~~~~~~~~~~~~
```

Comment:
I think we need to add a blank line after this one.

Reply:
Also the length to `~~~~~~~~~~~~~` must be exactly the length of the text above.

So this should be:

```
Specifying an elasticsearch index
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```